ivy/functional/frontends/paddle/tensor/math.py

```diff
@@ -408,6 +408,14 @@ def prod(x, axis=None, keepdim=False, dtype=None, name=None):
     return ivy.prod(x, axis=axis, keepdims=keepdim, dtype=dtype)
 
 
+@with_supported_dtypes(
+    {"2.5.1 and below": ("float32", "float64", "int32", "int64")}, "paddle"
```

Comment:
Is this correct? It is stated in the [floor_divide docs](https://www.paddlepaddle.org.cn/documentation/docs/en/api/paddle/floor_divide_en.html) that:
> x (Tensor) – the input tensor, it’s data type should be uint8, int8, int32, int64, float32, float64, float16, bfloat16.

Reply:
I have solved this issue.

Reply:
I updated the changes and pushed the files. Check it